docs/distribution/release-policy.md

```diff
@@ -132,6 +133,74 @@ git tag --sign enso-0.2.11
 17. Once verification has been performed, the release can be published. It
     should _not_ be a pre-releases as we reserve these for nightly builds.
 
+### Breaking Release Workflow
+
+If, however, the engine needs to release but the `HEAD` of `main` is not in a
+compatible state with the IDE, the process has to differ a little bit. Please
+note that the instructions here are more vague than the above, as exactly what
+is required may vary based on the situation.
+
+Consider a scenario where there are four new commits since the last release:
+`wwwwwww`, `xxxxxxx`, `yyyyyyy`, `zzzzzzz`. The commit `yyyyyyy` contains
+breaking changes that are _not yet integrated with the IDE_. Releasing a package
+containing that commit (and those that depend on it) would break the IDE, but we
+nevertheless want to release as much as possible:
+
+1.  If no release branch exists for the current major version, one should be
+    created.
+2.  Rebase the commits that are wanted onto the release branch:
+
+```
+git rebase --onto origin/release/0.x wwwwwww~1 xxxxxxx
+```
+
+3.  This will put you into a "detached HEAD" state at commit `zzzzzzz`, so you
+    need to make a new branch: `git branch release-update zzzzzzz`.
```

Comment:
This is not `zzz`. I'd clarify what exactly the newly created commit represents here.